Project.toml

```diff
@@ -9,13 +9,17 @@ ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FastGaussQuadrature = "442a2c76-b920-505d-bb47-c5924d526838"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 GPLikelihoods = "6031954c-0455-49d7-b3b9-3e1c99afaf40"
 KLDivergences = "3c9cd921-3d3f-41e2-830c-e020174918cc"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Optim = "429524aa-4258-5aef-a3af-852621145aeb"
+QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
Would be required by warmstart-procedure (both Laplace and EP) for `Zygote.ignore()`... should this be an optional dependency (using Required), as e.g. kernelkronmat for Kronecker.jl in KernelFunctions?

Reply:
Personally, I think we should just add an `ignore` function modelled on Zygote.ignore to ChainRulesCore, or just add it in this package using ChainRulesCore and an `rrule`. This would avoid the Zygote dependency entirely.

Reply:
It's as simple as
```julia
ignore(closure) = closure()
@non_differentiable ignore(closure)
```

Reply:
Opened https://github.com/JuliaDiff/ChainRulesCore.jl/issues/470